### PR TITLE
Rds/cml/10 7Updated

### PR DESCRIPTION
**Pull request by claperle**

### CODE REVIEW

File: docs/section10/10-7.md [hunk 1]
```diff
@@ -29,48 +29,45 @@
...
+                - Check else if the system type is 3, 3b and that the associated zone.equipment_count equal Null, this means the number of aggregated hvac zones was not defined (pass if efficiency meets a lower size category or the size category associated with the system). Assumes that the HVAC system serves one zone which is consistent with the rules of App G.:`elif (hvac_sys_type_b = "SYS-3" or hvac_sys_type_b = "SYS-3b") AND zone_list_b[0].equipment_count == Null:`
                     - Get baseline efficiency for the <65,000 Btuh size category:`COPnf_full_load_65Btuh_b = data_lookup(table_G3_5_1,65Btuh)`
                     - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_65Btuh_b: meets_lower_or_equiv_size_category_eff_b = TRUE`
-                    - Check if capacity is not in the <65,000 Btuh size category: `if if total_cool_capacity_b >= 65000:`
+                    - Check if capacity is greater or equal to 65000 and the efficiency is not equal to a lower size bracket efficiency: `if total_cool_capacity_b >= 65000 and meets_lower_or_equiv_size_category_eff_b != TRUE:`
```
Comment (mkarpman) [inline docs/section10/10-7.md:57]: As mentioned above, efficiency for capacity bracker over 760 kBtu/Hr is higer than for 65 Kbtu/Hr

File: docs/section10/10-7.md [hunk 2]
```diff
@@ -86,8 +83,9 @@
                     - Else capacity is >=760,000 Btuh: `Else:`
                         - Get baseline efficiency for the >=760,000 Btuh size category:`COPnf_full_load_760Btuh_b = data_lookup(table_G3_5_1,760Btuh)`
                         - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_760Btuh_b: meets_lower_or_equiv_size_category_eff_b = TRUE`  
-                
-                 - Check else if the system type is 4 and that num_zones_defined_for_all_zones equals TRUE (pass if efficiency meets size category associated with the system):`elif (hvac_sys_type_b = "SYS-3" or hvac_sys_type_b = "SYS-3b") AND num_zones_defined_for_all_zones == FALSE:`
+               
+                 - Check else if the system type is 4 and that the associated zone number of aggregated hvac zones was defined (pass if efficiency meets size category associated with the system):`elif (hvac_sys_type_b = "SYS-3" or hvac_sys_type_b = "SYS-3b") AND zone_list_b[0].equipment_count != Null:`
```
Comment (mkarpman) [inline docs/section10/10-7.md:87]: Code should reference system 4 and 4b, as mentioned in the charging note, not 3 & 3b.

File: docs/section10/10-7.md [hunk 3]
```diff
@@ -100,21 +98,21 @@
...
+                        - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_240Btuh_b: meets_lower_or_equiv_size_category_eff_b = TRUE`                   
                                                 
                 **Rule Assertion:**
-                - Case 1: if the modeled efficiency matches the expected baseline efficiency based on the size category or if for systems 3 and 4 the modeled efficiency matches the expected efficiency based on the size category or the efficiency of a smaller size category then pass: `if meets_lower_or_equiv_size_category_eff_b == TRUE: PASS`             
+                - Case 1: if the modeled efficiency matches the expected baseline efficiency based on the size category or if for systems 3 and 4 the modeled efficiency matches the expected efficiency based on the size category or the efficiency of a smaller size category and the efficiency was defined in the correct units then pass: `if meets_lower_or_equiv_size_category_eff_b == TRUE AND was_efficiency_found == TRUE : PASS`
```
Comment (mkarpman) [inline docs/section10/10-7.md:115]: Potentially have UNDETERMINED outcome if capacity is missing.

**Review by mkarpman — APPROVED** [review]

File: docs/section10/10-7.md (hunk 2 above)
Comment (claperle) [reply docs/section10/10-7.md:87]: Corrected.

File: docs/section10/10-7.md [hunk 4]
```diff
@@ -29,48 +29,45 @@
 - Get list of hvac systems of type 5 and 6 that serve more than one floor: `hvac_sys_serve_more_than_one_flr_list_b = get_hvac_systems_5_6_serving_multiple_floors_b(B_RMR).keys()`
 - Get dictionary list of baseline zones and the associated HVAC systems: `hvac_zone_list_w_area_dict_b = get_hvac_zone_list_w_area (B_RMR)`  
     - For each hvac system in the the B_RMR: `for hvac_b in B_RMR...HeatingVentilationAirConditioningSystem:`
-        - Reset boolean variable that indicates if the hvac system is type 3 or 4 (used in rule assertion): `sys_type_3_or_4 = FALSE`
-        - Reset boolean variable that indicates if the hvac system is in the smallest capacity bracket (used in rule assertion): `smallcapBracket = FALSE`
+        - Reset the meets_lower_or_equiv_size_category_eff_b boolean variable to FALSE: `meets_lower_or_equiv_size_category_eff_b = FALSE`
```
[the remark this answers is not in the document]
Comment (claperle) [reply docs/section10/10-7.md]: Per email, implemented logic such that if the aggregation_factor (new name) is undefined and the efficiency is consistent with the size category assuming no zone aggregation OR is a more conservative efficiency from a lower bracket and size breacket we pass it

File: docs/section10/10-7.md [hunk 5]
```diff
@@ -29,48 +29,45 @@
...
+            Loop through each cooling system (there will only be one) and get the attributes of the cooling system(s) associated with this HVAC (e.g. efficiency, capacity, etc.) and determine if minimum efficiency requirements are met via Appendix G  
+            - For each cooling_system_b in hvac_b: `for cooling_system_b in hvac_b.cooling_system:`  
+                - Check if rated cooling capacity does not equal null, if it does not equal null then set the cooling capacity equal to this data element's value: `if cooling_system_b.rated_total_cool_capacity <> Null:`
                     - Set cooling capacity equal to the rated cooling capacity: `total_cool_capacity_b = cooling_system_b.rated_total_cool_capacity`
                 - Else, rated equals Null then set the cooling capacity equal to the design cooling capacity:`total_cool_capacity_b = cooling_system_b.design_total_cool_capacity`
```
[the remark this answers is not in the document]
Comment (claperle) [reply docs/section10/10-7.md]: Addressed

File: docs/section10/10-7.md (hunk 1 above)
Comment (claperle) [reply docs/section10/10-7.md]: Per email, implemented logic such that if the aggregation_factor (new name) is undefined and the efficiency is consistent with the size category assuming no zone aggregation OR is a more conservative efficiency from a lower bracket and size breacket we pass it

File: docs/section10/10-7.md (hunk 3 above)
Comment (claperle) [reply docs/section10/10-7.md]: Addressed

**Review by mkarpman — APPROVED** [review]

File: docs/section10/10-7.md [hunk 6]
```diff
@@ -29,93 +29,106 @@
...
-                - Else, rated equals Null then set the cooling capacity equal to the design cooling capacity:`total_cool_capacity_b = cooling_system_b.design_total_cool_capacity`
-                - Get the full load no fan cooling efficiency: `full_load_efficiency_b = cooling_system_b.full_load_efficiency_no_fan`
-                - Reset meets size category or lower size category efficiency boolean variable: `meets_lower_or_equiv_size_category_eff_b = FALSE` 
-                Based upon the system type and cooling capacity determine the efficiency that is expected to be modeled
-                - Check if the system is system type 1 or 1b: `if hvac_sys_type_b = "SYS-1" or hvac_sys_type_b = "SYS-1b":`
-                    - Set COPnf_full_load_eff_b = 3.2:`COPnf_full_load_eff_b = 3.2`
-                    - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_eff_b: meets_lower_or_equiv_size_category_eff_b = TRUE`
+                - Else if, check that design cooling capacity does not equal null:`Elif cooling_system_b.design_total_cool_capacity <> Null:`
```
[the remark this answers is not in the document]
Comment (claperle) [reply docs/section10/10-7.md:44]: @mkarpman  Please clarify, I do not understand why the if should be deleted.

File: docs/section10/10-7.md [hunk 7]
```diff
@@ -29,93 +29,106 @@
 - Get list of hvac systems of type 5 and 6 that serve more than one floor: `hvac_sys_serve_more_than_one_flr_list_b = get_hvac_systems_5_6_serving_multiple_floors_b(B_RMR).keys()`
 - Get dictionary list of baseline zones and the associated HVAC systems: `hvac_zone_list_w_area_dict_b = get_hvac_zone_list_w_area (B_RMR)`  
     - For each hvac system in the the B_RMR: `for hvac_b in B_RMR...HeatingVentilationAirConditioningSystem:`
-        - Reset boolean variable that indicates if the hvac system is type 3 or 4 (used in rule assertion): `sys_type_3_or_4 = FALSE`
-        - Reset boolean variable that indicates if the hvac system is in the smallest capacity bracket (used in rule assertion): `smallcapBracket = FALSE`
+        - Reset the efficiency_matches_acceptable_value boolean variable to FALSE: `efficiency_matches_acceptable_value = FALSE`
```
[the remark this answers is not in the document]
Comment (claperle) [reply docs/section10/10-7.md:33]: Addressed

**Review by supriyagoel — APPROVED** [review]

**Pull request merged**